packages/conf-sed/conf-sed.1.0/opam

```diff
@@ -0,0 +1,23 @@
+opam-version: "2.0"
+maintainer: "unixjunkie@sdf.org"
+homepage: "https://www.gnu.org/software/sed/"
+authors: "https://www.gnu.org/software/sed/#maintainers"
+bug-reports: "https://github.com/ocaml/opam-repository/issues"
+license: "GPL-3.0-or-later"
+build: ["which" "sed"]
+depends: ["conf-which"]
+depexts: [
+  ["sed"] {os-family = "debian"}
+  ["sed"] {os-distribution = "centos"}
+  ["sed"] {os-distribution = "fedora"}
+  ["sed"] {os-distribution = "ol"}
+  ["sed"] {os = "macos" & os-distribution = "homebrew"}
+  ["sed"] {os-distribution = "alpine"}
+  ["sed"] {os-family = "suse"}
+  ["sed"] {os-family = "arch"}
+  ["sed"] {os = "freebsd"}
```

Comment:
there's no such package on FreeBSD. The BSD `sed` is shipped with the base system, and there is a package named `gsed` which installs a `GNU sed` (the installed binary is `gsed`) -- I'd recommend to remove the above line from this conf package.

Reply:
fixed